docs/introduction.md

```diff
@@ -42,7 +44,34 @@ The final version of the previous execution state remains available through a le
 
 ## Flow
 
-Flow was designed to provide a blockchain that can scale while preserving composability in a single blockchain state. This is achieved by introducing several [node roles](#nodes), each with their own trade-off for the [scalability trilemma](https://aakash-111.medium.com/the-scalability-trilemma-in-blockchain-75fb57f646df).
+Most current blockchains are built as a homogenous system comprised of *full nodes*.
+Each full node is expected to collect and choose the transactions to be included in the next block, execute the block, come to a consensus over the output of the block with other full nodes, and finally sign the block and append it to the chain.
+
+Existing blockchain architectures have well documented throughput limitations. There are two main approaches to combat this:
+
+1. Layer-2 networks
+2. Sharding
+
+Layer-2 solutions include include networks like Bitcoin's Lightning and Ethereum's Plasma — these networks work off the main chain.
+
+Sharding is a technique where a network is broken up into many interconnected networks.
+Sharding significantly increases the complexity of the programming model by breaking ACID guarantees (Atomicity, Consistency, Isolation and Durability), increasing the cost and time for application development.
+
+Flow was designed to provide a blockchain that can scale while preserving composability in a single blockchain state.
+This is achieved by a novel approach where work traditionally assigned to full nodes is split and assigned to specific roles, allowing pipelining.
+We recognize the following roles in the Flow architecture.
+
+1. Collection role — in charge of transaction collection from the user agents
+2. Execution role — in charge of executing the transactions
+3. Consensus role — maintain the chain of blocks and responsible for the chain extension by appending new blocks. These nodes also rule on reported misbehaviors of other nodes
+4. Verification role — done by a more extensive set of verification nodes, they confirm that the execution results are correct
+5. Access role — also called Observer role — includes nodes that relay data to protocol-external entities that are not participating in the protocol
```

Comment:
I wonder how much of this you just copied? Or is this in your own words? Whenever you just copy passages, I would suggest you summarize them instead and link to the original documents.

Reply:
This one is pretty similar in the original docs, I think I did copy it and rephrase a bit, and arranged the node roles in a list rather than a paragraph. I felt it provided a nice comparison/summary of the different roles.

Reply:
Should I remove this, what's the ruling?